stacker/providers/aws.py

```diff
@@ -0,0 +1,98 @@
+import logging
+
+import boto
+from boto import cloudformation
+
+from .base import BaseProvider
+
+logger = logging.getLogger(__name__)
+
+
+class Provider(BaseProvider):
```

Comment:
Same comment as with Actions - should this be called AWSProvider to make it more explicit/easier to read?